quickstarts/elixir.mdx

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Elixir'
+description: 'Learn how to use Novu to quickly send multi-channel (SMS, Email, Chat, Push) notifications with Elixir.'
+---
+
+Quickly get started with the integration of Novu into your Elixir project.
+Novu is an open-source application that facilitates the delivery of embellished notifications across channels such as SMS, e-mail, push notifications, etc. This guide will help you integrate these notifications into your Elixir project.
+
+Let's get into it! 🚀
+
+# Requirements
+
+To be able to use this Quickstart, you would need to have the following pre-requisite:
+
+- A Novu account; if you are new and don't have one, sign up on (https://web.novu.co/) for free.
+- A working Elixir project.
+
```

Comment:
There should be a step to Install and Set up the Novu Elixir SDK in your project